src/api.ts

```diff
@@ -0,0 +1,16 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
+
+const getItems = async<T>(url: string): Promise<T[]> => {
+  const response = await fetch(API_URL + url);
```

Comment:
We should make a distinction between the base URL and the resource we are calling. It is better to use template literals for this.

```suggestion
  const response = await fetch(`${BASE_API_URL}/${resourceUrl});
```